packages/react-strict-dom/src/types/StrictReactDOMProps.js

```diff
@@ -281,6 +281,7 @@ export type StrictReactDOMProps = $ReadOnly<{
 
   // Other
   autoCapitalize?: ?(
+    | true
     | 'off'
     | 'none'
     | 'on'
```

Comment:
Potentially we could remove `off` and `on` as they were part of the pre-standard, iOS version of autoCapitalize and were retained for backwards compatibility. Not necessary for now, just putting it out there.